mod/consensus/pkg/cometbft/service/service.go

```diff
@@ -115,11 +114,9 @@ func NewService[
 		s.sm.CommitMultiStore().SetInterBlockCache(s.interBlockCache)
 	}
 
-	// Load the s.
-	if loadLatest {
-		if err := s.sm.LoadLatestVersion(); err != nil {
-			panic(err)
-		}
+	// Load latest height, once all stores have been set
+	if err := s.sm.LoadLatestVersion(); err != nil {
+		panic(err)
```

Comment:
we must do this, otherwise beconKit will panic. The `if loadLatest` is confusing to me, as it suggest this initialization is not mandatory

Reply:
I think its safe to remove, I think loadLatest was for if u want to instantiate a the cosmos BaseApp without initializing the DB, I don't see why we need this tho.